style.css

```diff
@@ -135,11 +161,285 @@ body {
+    background-color: #0056ec;
+  }
+
+  .menu__btn > span::before {
+    content: '';
+    top: -8px;
+  }
+
+  .menu__btn > span::after {
+    content: '';
+    top: 8px;
+  }
+
+  .menu__box {
+    display: block;
+    position: fixed;
+    visibility: hidden;
+    top: 0;
+    width: 375px;
+    height: 100%;
+    margin: 0;
+    padding: 86px 0;
+    text-align: center;
+    background-color: #fff;
+  }
+
+  .menu__item {
+    display: block;
+    font-family: "Roboto", arial, helvetica, sans-serif;
+    font-weight: 500;
+    top: 79px;
+    padding: 15px;
+    margin: 10px;
+    font-size: 29px;
+  }
+
+  .menu__item:nth-child(1) {
+    padding: 18px;
+    margin-right: 9px;
+  }
+
+  .menu__item:nth-child(2) {
+    padding: 10px;
+    margin-right: 12px;
+  }
+
+  .menu__item:nth-child(3) {
+    padding: 19px 0 3px 0;
+    margin-right: 14px;
+  }
+
+  .menu__item:nth-child(4) {
+    padding: 27px;
+    margin-right: 18px;
+  }
+
+  .menu__item:nth-child(5) {
+    padding: 2px;
+    margin-right: 17px;
+  }
+
+  #menu__toggle:checked ~ .menu__btn > span {
+    transform: rotate(45deg);
+  }
+
+  #menu__toggle:checked ~ .menu__btn > span::before {
+    top: 0;
+    transform: rotate(0);
+  }
+
+  #menu__toggle:checked ~ .menu__btn > span::after {
+    top: 0;
+    transform: rotate(90deg);
+  }
+
+  #menu__toggle:checked ~ .menu__box {
+    visibility: visible;
+    left: 0;
+  }
+}
+
+@media (min-width: 769px) and (max-width: 1024px) {
+  .wrapper {
+    width: 91%;
+  }
+
+  .header {
+    margin-bottom: 66px;
+  }
+
+  .menu__item {
+    font: 500 14px/16px "Roboto", arial, helvetica, sans-serif;
+  }
+
+  .menu__item:nth-child(1) {
+    margin-right: 21px;
+  }
+
+  .menu__item:nth-child(2) {
+    margin-right: 20px;
+  }
+
+  .menu__item:nth-child(3) {
+    margin-right: 23px;
+  }
+
+   .menu__item:nth-child(4) {
+     margin-right: 24px;
+  }
+
+  .menu__item:last-child {
+    margin-right: 0;
+  }
+
+  .main {
+    padding: 35px 50% 126px 0;
+    background: top right no-repeat url(img/mainImg1.svg);
+    background-position-y: 20px;
+  }
+
+  .main__title {
+    margin: 0 0 38px 0;
+    font: bold 50px/66px "pt_serif", arial, helvetica, sans-serif;
+  }
+
+  .main__subtitle {
+    margin: 0 0 40px 0;
+    font: 500 18px/22px "Roboto", arial, helvetica, sans-serif;
+    max-width: 350px;
+  }
+
+  .main__button {
+    font: 500 14px/16px "Roboto", arial, helvetica, sans-serif;
+    padding: 13px 20px;
+    margin: auto;
+  }
+}
+
+@media (min-width: 1025px) and (max-width: 1366px) {
+  .wrapper {
+    width: 68%;
+  }
+
+  .header {
+    margin-bottom: 66px;
+  }
+
+  .menu__item {
+    font: 500 14px/16px "Roboto", arial, helvetica, sans-serif;
+  }
+
+  .menu__item:nth-child(1) {
+    margin-right: 21px;
+  }
+
+  .menu__item:nth-child(2) {
+    margin-right: 20px;
+  }
+
+   .menu__item:nth-child(3) {
+     margin-right: 23px;
+  }
+
+   .menu__item:nth-child(4) {
+     margin-right: 24px;
+  }
+
+   .menu__item:last-child {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/new_project2.0/pullRequest?prid=6297664)